examples/cassandra/README.md

```diff
@@ -364,9 +452,18 @@ Create this daemonset:
 $ kubectl create -f examples/cassandra/cassandra-daemonset.yaml
 ```
 
-Now if you list the pods in your cluster, and filter to the label ```name=cassandra```, you should see one cassandra pod for each node in your network:
+Make sure that your Kubernetes cluster has enabled use of the experimental API.
```

Comment:
This mention of the experimental api will be outdated in 1.2, so maybe I shouldn't even add it here.